### DISCUSSION TITLE
Chat is visible for Admins?

### DISCUSSION THREAD

**Discussion by oschmidteu** [open-webui/open-webui#5213] [Q&A] [answered]
Why is it possible for admins to view user chats on Open WebUI, and how can this be prevented or restricted?
Or can it preventet by using this variable.:
SHOW_ADMIN_DETAILS
Type: bool
Default: True
Description: Toggles whether to show admin user details in the interface.

**Comment by silentoplayz** [comment, marked as answer]
The super-admin account (the first sign-up of your Open WebUI instance) can disable the visibility of other user chats in the admin panel of Open WebUI by setting the environment variable `ENABLE_ADMIN_CHAT_ACCESS=false`. [Here](https://github.com/open-webui/open-webui/pull/4351) is the related merged PR, for reference.

**Reply by oschmidteu** [reply]
Thanks a lot. I didn't saw that in the docs (I guess the PR is yet to new for the docs).

**Comment by Chopper7729** [comment]
Hello! Im very new to this! How can i add the variable when using docker? Is it the full docker run commnad? Then do i use the export command with the variable to set it? Thanks!

**Reply by ihatenodejs** [reply]
You might have figured it out by now, but you can pass the environment variable to the Docker container as well. This way, it persists through restarts of the container too.

In Docker Compose:
```yaml
services:
  open-webui:
    ...
    environment:
      - ENABLE_ADMIN_CHAT_ACCESS=false
```

In Docker Run:
```
docker run -e ENABLE_ADMIN_CHAT_ACCESS=false ...
```
